ironfish/src/node.ts

```diff
@@ -205,6 +205,7 @@ export class IronfishNode {
     const telemetry = new Telemetry({
       chain,
       logger,
+      graffiti: config.get('blockGraffiti'),
```

Comment:
Should we pass in graffiti and get this adhoc incase they change their graffiti while the node is running? Otherwise they'll have to restart the node.

Reply:
Agreed with @NullSoldier